src/utils/loaders/google_map_loader.js

```diff
@@ -62,8 +62,12 @@ export default function googleMapLoader(bootstrapURLKeys) {
       ''
     );
 
+    const url = bootstrapURLKeys.region === 'cn'
```

Comment:
what about other regions ?

Reply:
@jooj123 I believe the only region that has a problem with this is China. But we could confirm

Reply:
Exactly, China is the only region mentioned on the Google Maps FAQ (see link in the issue), and the only region I've experienced problems with.